Makefile

```diff
@@ -181,7 +181,7 @@ batcher_fund_service_manager_balance:
 ./batcher/aligned-batcher/.env:
 	@echo "To start the Batcher ./batcher/aligned-batcher/.env needs to be manually set"; false;
 
-batcher_start: ./batcher/aligned-batcher/.env batcher_fund_service_manager_balance
+batcher_start: ./batcher/aligned-batcher/.env
```

Comment:
I think we should also delete the whole `make batcher_fund_service_manager_balance` target